ext/standard/basic_functions.c

```diff
@@ -643,11 +643,9 @@ PHP_FUNCTION(long2ip)
 	ip = (zend_ulong)sip;
 
 	myaddr.s_addr = htonl(ip);
-	if (inet_ntop(AF_INET, &myaddr, str, sizeof(str))) {
-		RETURN_STRING(str);
-	} else {
-		RETURN_FALSE;
-	}
+	inet_ntop(AF_INET, &myaddr, str, sizeof(str));
```

Comment:
I think what @nielsdos wanted is more something like

```
const char *ret = inet_ntop(AF_INET, &myaddr, str, sizeof(str));
ZEND_ASSERT(ret != NULL);
RETURN_STRING(str);
```

Reply:
That's what I meant indeed, as Jorg added the assertion, but I don't care too much about this.

Reply:
that would also, in turn, avoid the pedantic build warning `please care about return values`-like

Reply:
Thanks for the hint. I have added back the assertion.